pydualsense/pydualsense.py

```diff
@@ -12,12 +12,56 @@
 from .event_system import Event
 from copy import deepcopy
 
+import array
 
 logger = logging.getLogger()
 FORMAT = '%(asctime)s %(message)s'
 logging.basicConfig(format=FORMAT)
 logger.setLevel(logging.INFO)
 
+hashTable = array.array('I', [
```

Comment:
please transfer the whole crc calculations to a seperate file.

Reply:
No problem